projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -0,0 +1,74 @@
+<ng-container *tuiLet="options.metadata | async as phoneMetadata">
+    <div
+        *ngIf="countriesNames$ | async as countriesNames"
+        tuiGroup
+        [tuiDropdown]="dropdown"
+        [tuiDropdownEnabled]="!readOnly"
+        [(tuiDropdownOpen)]="open"
+        (tuiActiveZoneChange)="onActiveZone($event)"
+    >
+        <tui-textfield
+            tuiChevron
+            class="t-country-select tui-group__auto-width-item"
+            [content]="flag"
+        >
+            <select
+                tuiTextfield
+                [disabled]="disabled"
+                [readOnly]="readOnly"
+            ></select>
+
+            <ng-template #flag>
+                <img
+                    *ngIf="countryIsoCode$ | async as countryIsoCode"
+                    class="t-flag"
+                    [alt]="countriesNames[countryIsoCode]"
+                    [src]="countryIsoCode | tuiFlag"
+                />
+            </ng-template>
+        </tui-textfield>
+
+        <tui-primitive-textfield
```

Comment:
Why not new textfield, since you already got familiar enough with it to use it for country selection? :D

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/314477590a0bd9455a5719cbd3db2ea2cb6ca916